docs/details/widgets/buttonmatrix.rst

```diff
@@ -50,54 +55,67 @@ width calculated automatically. So in the example the first row will
+The buttons' width can be set in proportion to the width of other buttons in the same
+row with :cpp:expr:`lv_buttonmatrix_set_button_width(btn_matrix, button_id, width)` E.g. in a
 line with two buttons: *buttonA, width = 1* and *buttonB, width = 2*, *buttonA*
-will have 33 % width and *buttonB* will have 66 % width. It's similar to
-how the
+will have 33 % width and *buttonB* will have 66 % width.  This is similar to how the
 `"flex-grow" <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow>`__
-property works in CSS. The width must be in the [1..15] range and the
-default width is 1.
+property works in CSS. The width must be in the range [1..15] with the default being 1.
 
-In addition to the width, each button can be customized with the
-following parameters:
+Button behavior
+---------------
+
+Each button's behavior can be customized with the following control flags:
 
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_HIDDEN`: Makes a button hidden (hidden buttons still take up space in the layout, they are just not visible or clickable)
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_NO_REPEAT`: Disable repeating when the button is long pressed
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_DISABLED`: Makes a button disabled Like :cpp:enumerator:`LV_STATE_DISABLED` on normal Widgets
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CHECKABLE`: Enable toggling of a button. I.e. :cpp:enumerator:`LV_STATE_CHECKED` will be added/removed as the button is clicked
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_HIDDEN`: Hides button; it continues to hold its space in layout.
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_NO_REPEAT`: Do not emit LV_EVENT_LONG_PRESSED_REPEAT events while button is long-pressed.
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_DISABLED`: Disables button like LV_STATE_DISABLED on normal Widgets.
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CHECKABLE`: Enable toggling of LV_STATE_CHECKED when clicked.
 - :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CHECKED`: Make the button checked. It will use the :cpp:enumerator:`LV_STATE_CHECHKED` styles.
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CLICK_TRIG`: Enabled: send LV_EVENT_VALUE_CHANGE on CLICK, Disabled: send :cpp:enumerator:`LV_EVENT_VALUE_CHANGE` on PRESS
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_POPOVER`: Show the button label in a popover when pressing this key
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CUSTOM_1`: Custom free to use flag
-- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CUSTOM_2`: Custom free to use flag
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CLICK_TRIG`: 1: Enables sending LV_EVENT_VALUE_CHANGE on CLICK, 0: sends LV_EVENT_VALUE_CHANGE on PRESS.
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_POPOVER`: Show button text in a pop-over while being pressed.
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_RECOLOR`: Enable text recoloring with `#color`
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CUSTOM_1`: Custom free-to-use flag
+- :cpp:enumerator:`LV_BUTTONMATRIX_CTRL_CUSTOM_2`: Custom free-to-use flag
```

Comment:
Amazing as usual. Just one thing.

"LV_EVENT_LONG_PRESSED_REPEAT" could become ":cpp:enumerator:\`LV_EVENT_LONG_PRESSED_REPEAT\`" and others.

\`#color\` could become \`\`#color\`\`.

Reply:
@liamHowatt @kisvegabor Another nice catch!  Both are fixed now and are reflected here:

http://crystal-clear-research.com/for_gabor/demo/docs.lvgl.io/master/details/widgets/buttonmatrix.html

Reply:
Thanks! That's what I meant. However there are some others ones in this highlighted block that need the same treatment. They are:

- LV_STATE_DISABLED
- LV_STATE_CHECKED
- LV_EVENT_VALUE_CHANGE

And perhaps CLICK can be \`\`CLICK\`\` and PRESS \`\`PRESS\`\`

Reply:
> Thanks! That's what I meant. However...

Agreed!  I finally set up my editor to do reStructuredText highlighting so that these things pop out at me better.  I found (and fix) 1 or 2 more as well.

Reply:
Thanks!!